### PR TITLE
Update the # public providers estimate in the Terraform Intro

### PR DESCRIPTION
**Pull request by brandonc**
This number has grown rapidly. [ref](https://registry.terraform.io)

### CODE REVIEW

**Comment by apparentlymart** [comment]
Hi @brandonc!

Just as an explanation for why you're seeing some radio silence here: this "intro" part of the docs largely belongs to the marketing folks rather than being a typical part of the documentation. What you proposed here seems reasonable to me but it's not typically my place to approve changes to this content. I'm not sure who is actually responsible for doing that.

With that said, this makes me wonder if it would be better to revise this text to be more "evergreen" by not mentioning a number at all. Of course that only exacerbates the "not clear who should approve it" situation, so probably better to save that for separate work rather than try to do it in this PR, but I'm mentioning it mainly because hopefully someone who is allowed to make that decision will see this PR and can consider it separately. :grinning:

**Review by laurapacilio — APPROVED** [review]
This looks right to me!! @gautambaghel do you have any insight into who from our marketing team could weigh in on the number here? I agree with @apparentlymart that it would be good to get buy-in from those folks as well.

For what it's worth, I think having the number here helps illustrate how comprehensive the registry is. but I do understand it's not evergreen :p

File: website/docs/intro/index.mdx
Comment (laurapacilio) [inline website/docs/intro/index.mdx:19]:
```suggestion
HashiCorp and the Terraform community have already written **more than 2500 providers** to manage thousands of different types of resources and services, and this number continues to grow. You can find all publicly available providers on the [Terraform Registry](https://registry.terraform.io/), including Amazon Web Services (AWS), Azure, Google Cloud Platform (GCP), Kubernetes, Helm, GitHub, Splunk, DataDog, and many more.
```

Comment (laurapacilio) [reply website/docs/intro/index.mdx:19]: @brandonc it's even higher now! :)

Comment (laurapacilio) [reply website/docs/intro/index.mdx:19]: okay I like this - it's definitely correct and definitely more evergreen. Please add this in and then merge! P.S. I'm adding a backport label to your PR. After you merge this, please also merge the auto-generated backport label. This will make sure this change is reflected in our most recent docs version!

**Pull request merged**
